### PR TITLE
FDG-7769 General Site Updates - Make Dropdown Search height flexible for < 5 menu items

### PR DESCRIPTION
**Pull request by wyattch**
Ticket: https://federal-spending-transparency.atlassian.net/browse/FDG-7769
Coverage: 89.85%

### CODE REVIEW

File: src/components/combo-select/combo-currency-select/combo-select-dropdown/combo-select-dropdown.jsx [hunk 1]
```diff
@@ -112,11 +112,27 @@ const ComboSelectDropdown = ({
     }
   }, [active]);
 
+  const dropDownHeight = numOfItems => {
```
Comment (jdach) [inline src/components/combo-select/combo-currency-select/combo-select-dropdown/combo-select-dropdown.jsx:115]: This styling can be made dynamic. 
From testing in browser, I believe you can just conditionally apply a height to the dropdownList class when there are more than 5 items and add padding to the bottom of dropdownContainer

**Review by jdach — CHANGES_REQUESTED** [review]
Not sure how this is related to just a height change, but the search bar is disappearing after clearing out the search. I am unable to reproduce this in qat or prod. 
https://github.com/fedspendingtransparency/fiscal-data/assets/77741207/f403cd4a-9410-468c-b34b-b6d58245c1b5

**Comment by wyattch** [comment]
I was never able to replicate that search bar disappearing bug locally, so perhaps these dynamic changes will help?

**Review by hmargy — APPROVED** [review]
changes look good

**Pull request merged**